src/metabase/sync/sync_metadata/fields/sync_metadata.clj

```diff
@@ -73,6 +75,12 @@
                           old-database-type
                           new-database-type))
            {:database_type new-database-type})
+         (when (not= old-json-unfolding new-json-unfolding)
+           (log/info (trs "JSON unfolding setting of {0} has changed from ''{1}'' to ''{2}''."
+                          (common/field-metadata-name-for-logging table metabase-field)
+                          old-json-unfolding
+                          new-json-unfolding))
+           {:database_type new-database-type})
```

Comment:
```suggestion
           {:json_unfodling new-json-folding})
```

Reply:
and don't forget to add a test for this in `metabase.sync.sync-metadata.fields.sync-metadata-test`

Reply:
Fixed! https://github.com/metabase/metabase/commit/d51430fdbccf27e1597f6ae5c003ca6cb403f5d7